course_content/iris_course/1.The_Iris_Cube.ipynb

```diff
@@ -206,8 +208,9 @@
    "metadata": {},
    "source": [
     "<div class=\"alert alert-block alert-warning\">\n",
-    "    <b><font color='brown'>Exercise: </font></b> <p>What are the dimensions of the \"total electron content\" cube?</p>\n",
-    "    <p>What are the units of the \"electron_density\" cube?</p>\n",
+    "    <b><font color='brown'>Exercise: </font></b>\n",
+    "    <p>What are the dimensions of the \"total electron content\" cube?\n",
+    "    <br>What are the units of the \"electron_density\" cube?</p>\n",
```

Comment:
The difference between `<p></p>` and `<br>` is more obvious in jupyter lab than jupyter notebook.

I think `<br>` looks better.

Reply:
Ok !